catalog/spatial/csw/spatial-csw-endpoint/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/endpoint/mappings/CswRecordMapperFilterVisitor.java

```diff
@@ -79,6 +83,14 @@
 
     private Filter visitedFilter;
 
+    public CswRecordMapperFilterVisitor() {
```

Comment:
Looks like this isn't needed anymore since every should be passing the filterBuilder